### DISCUSSION TITLE
Weird image error - but vlc stream is running fine

### DISCUSSION THREAD

**Discussion by bib2** [blakeblackshear/frigate#13244] [General]
Hi, i installed frigate 0.14 on proxmox lxc with the ttech helper-script. It is working, i integrated my dahua webcam and my doorphone integrated cam (rtsp-stream).

But i get picture errors. Nearly grey, without color and some areas are pink and green. Also there is a line (or triangle) from lower left to upper right. --- The provided example-stream is working without problems. Also when i watch my streams inside vlc or any other app, they are working fine.

Any idea what could be the problem?

I used nearly basic config. For testing no iGPU passthrough and no coral. Only CPU-Power.

Here is my screenshot:

![frigate](https://github.com/user-attachments/assets/a5d6a617-5345-4c69-b882-f2567b8537e2)

If you need more infos, i will provide it. But maybe this is a known problem???

**Comment by hawkeye217** [comment]
See the [FAQ](https://github.com/blakeblackshear/frigate/discussions/12724), Usage question 15.

**Comment by bib2** [comment]
I doublechecked this before and i tested all available substreams and the webcam-webinterface.

Now i´m on substream_0 and full-hd resolution 1920x1080. And my detect width and hight are the same.

I also tested all other substreams and entered the right resolution. Always the same.

-----

But this is only the answer for the triangle? What about the bad picture quality/imaghe errors?

**Reply by hawkeye217** [reply]
Yes,the only reason for the distorted image (both triangle and colors) is a resolution mismatch. 

Some cameras wrongly report their resolution, sometimes it is off by just a few pixels. If you cannot find the correct resolution, the solution is to set up go2rtc as the FAQ indicated.

**Comment by bib2** [comment]
I installed a webcam viewer on my android phone and checked it again - you are right, the real resolution is 1088 x 1920 instead of 1080...

Thanks for your fast help!

**Reply by hawkeye217** [reply]
Great! Happy to help.
